django/views/decorators/cache.py

```diff
@@ -26,34 +25,23 @@ def cache_page(timeout, *, cache=None, key_prefix=None):
 
 
 def cache_control(**kwargs):
```

Comment:
`cache_control` and `never_cache` don't actually use `request` and the change https://github.com/django/django/pull/14030 should probably not have been added.